cmd/kubectl-testkube/commands/tests/run.go

```diff
@@ -59,21 +59,26 @@ func NewRunTestCmd() *cobra.Command {
 
 			printExecutionDetails(execution)
 
-			if watchEnabled {
-				watchLogs(execution.Id, client)
-			}
+			if execution.Id != "" {
```

Comment:
I think it's an error when there is no execution ID - we shouldn't be silent here.

Reply:
It works differently for scheduled tests. We don't have execution id after starting scheduled test. It will be just planned for execution and id will be created when scheduler triggers this test